### ISSUE TITLE
Unable to boot games with a dead battery

### ISSUE THREAD

**Issue by meyp58** [tpunix/SAROO#193]
Hi, first of all I want to apologize in advance if the following is already a known issue... However, I couldn't find anything about it. 

I was unable to boot any games with my Saroo. Every attempt to boot a game got stuck in the game selection menu with the message "Game Booting". I tried many different games from different sources but the result was always the same. After nothing I've found on the internet helped, the one thing was left I hadn't tried yet: renewing the dead battery of the Saturn. And voilà, the games startet booting. 

Long story short: I looks like the Saroo is unable to boot any games with a dead battery. 

Just wanted to let you know.

Thanks!

**Comment by tzmwx** [comment]
Please test again: reinstall a dead battery and reproduce the above situation

**Comment by meyp58** [comment]
Allright, case closed... After reinserting the dead battery the games still booted. Can you tell me, what might be the background of this? Was your feedback based on "general distrust" or is there a technical explanation for this? I can assure you, that I've struggled for days now and - as last resort - changing the battery instantly helped...
